DayTrader.xcodeproj/xcuserdata/jpun.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure why this .plist file is being changes.  Did you do something to another file beyond changing background color of the view?